kratos/processes/structured_mesh_generator_process.cpp

```diff
@@ -192,14 +215,15 @@ void StructuredMeshGeneratorProcess::GenerateNodes2D(Point const& rMinPoint, Poi
 void StructuredMeshGeneratorProcess::GenerateNodes3D(Point const& rMinPoint, Point const& rMaxPoint)
 {
     GeometryType::CoordinatesArrayType local_element_size = rMaxPoint - rMinPoint;
-    local_element_size /= mNumberOfDivisions;
-    Point local_coordinates = rMinPoint;
+    local_element_size[0] /= mNumberOfDivisions[0];
+    local_element_size[1] /= mNumberOfDivisions[1];
+    local_element_size[2] /= mNumberOfDivisions[2];    Point local_coordinates = rMinPoint;
```

Comment:
new line

Reply:
That was my first draft, but I didn't like the type changes between double and ints for the array.

Reply:
ping @roigcarlo